includes/Modules/Sign_In_With_Google.php

```diff
@@ -421,11 +433,62 @@ private function can_render_signinwithgoogle() {
 	 */
 	private function render_button_in_wp_login_form( $content ) {
 		if ( $this->can_render_signinwithgoogle() ) {
-			$content .= '<div class="googlesitekit-sign-in-with-google__frontend-output-button"></div>';
+			ob_start();
+			/**
+			 * Display the Sign in with Google button.
+			 *
+			 * @since n.e.x.t
+			 *
+			 * @param array $args Optional arguments to customize button attributes.
+			 */
+			do_action( 'googlesitekit_render_sign_in_with_google_button' );
+			$content .= ob_get_clean();
 		}
 		return $content;
 	}
 
+		/**
+		 * Renders the Sign in with Google button markup.
+		 *
+		 * @since n.e.x.t
+		 *
+		 * @param array $args Optional arguments to customize button attributes.
+		 */
+	public function render_sign_in_with_google_button( $args = array() ) {
+		if ( ! is_array( $args ) ) {
+			$args = array();
+		}
+
+		$default_classes = array( 'googlesitekit-sign-in-with-google__frontend-output-button' );
+		$passed_classes  = array();
+		if ( ! empty( $args['class'] ) ) {
+			$passed_classes = is_array( $args['class'] ) ? $args['class'] : preg_split( '/\s+/', (string) $args['class'] );
+		}
+
+		$classes = array_map( 'sanitize_html_class', array_merge( $default_classes, $passed_classes ) );
```

Comment:
"Passed" is vague, let's go with something like `$classes_from_args` 🤔 

```suggestion
		$classes_from_args  = array();
		if ( ! empty( $args['class'] ) ) {
			$classes_from_args = is_array( $args['class'] ) ? $args['class'] : preg_split( '/\s+/', (string) $args['class'] );
		}

		$classes = array_map( 'sanitize_html_class', array_merge( $default_classes, $classes_from_args ) );
```